inception-kb/src/main/java/de/tudarmstadt/ukp/inception/kb/querybuilder/SPARQLQueryBuilder.java

```diff
@@ -187,41 +218,168 @@ public Iri getDescriptionProperty(KnowledgeBase aKb) {
             }
         }
         
-        public GraphPattern scopePattern(KnowledgeBase aKB, Iri aScope)
+        public GraphPattern descendentsPattern(KnowledgeBase aKB, Iri aContext)
```

Comment:
Javadoc pls

Reply:
I'm adding a reference to the place where the relevant documentation is.